pkgs/development/tools/ocaml/opam2json/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, lib, opam-installer, ocamlPackages }:
```

Comment:
Nit: the usual format these days is `lib` first, then other names in order of usage.